task-manager/src/app/task-create/task-create.component.scss

```diff
@@ -0,0 +1,11 @@
+#create-task-btn {
+  margin-left: 1vw;
+  width: 10vw;
```

Comment:
not sure if it is good idea to map button dimensions to viewport. At some point, button text won't fit the button